test-cases/manager/manager-regression-multiDC-large-set-distro.yaml

```diff
@@ -0,0 +1,27 @@
+test_duration: 1440
+
+prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=550100150 -schema 'replication(factor=3)' -mode cql3 native connectionsPerHost=64 -rate threads=500 throttle=45000/s -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..550100150",
```

Comment:
@ShlomiBalalis why are you using throttle here?
It's slowing down the prepare.

Also, Are you trying to write 1TB? why not splitting to 4 loaders and writing faster.

Reply:
This PR is outdated, I'll update it shortly